### ISSUE TITLE
Increase sensor resolution for temp + humidity to 0.01

### ISSUE THREAD

**Issue by Nab0y** [devbis/z03mmc#17]
Is it possible for the temperature and humidity data to be rounded to the nearest hundredth of a percent? For example, not 21.3, but 21.38 and so on.

**Comment by Mostalk** [comment]
In my case humidity only in integer. Where float values on chart it's BLE firmware, then Zigbee. 

![image](https://github.com/devbis/z03mmc/assets/9198186/dd9ff0db-af27-4719-a2c4-6214a0f9f2e6)


I try changing this values, but nothing happened 

![image](https://github.com/devbis/z03mmc/assets/9198186/89e69a30-fa15-4fa2-aa94-7d6a63b0f52d)

Temperature always have one value after the decimal point, but humidity doesn’t have this value at all

```
{
    "battery": 61,
    "humidity": 56,
    "last_seen": "2023-10-22T20:59:53.477Z",
    "linkquality": 200,
    "temperature": 23.5,
    "update": {
        "installed_version": 268578817,
        "latest_version": 268578817,
        "state": "idle"
    },
    "voltage": 2600
}

```

**Comment by devbis** [comment]
It is fixed in release 1.0.3

**Comment by Mostalk** [comment]
Great work. Thanks.
